code/modules/events/rogue_drones.dm

```diff
@@ -10,12 +10,7 @@
 		if(C.name == "carpspawn")
 			possible_spawns.Add(C)
 
-	//25% chance for this to be a false alarm
-	var/num
-	if(prob(25))
-		num = 0
-	else
-		num = rand(2,6)
+	var/num = rand(4,10)
```

Comment:
4 drones minimum is an ass for a moderate event.

Down to 2, leave the upper limit.